src/components/Pantry/Pantry.css

```diff
@@ -0,0 +1,20 @@
+.pantry {
+    margin: 0px 0px 0px 0px;
+    background-color: azure;
+}
+
+.pantry-heading {
+    padding: 1rem;
+    margin: auto;
+
+}
+
+.product-form {
+    margin-left: 1rem;
+    margin-right: 1rem;
+}
+
+.product-submit {
+    align-self: flex-end;
+    margin-bottom: 17px;
```

Comment:
It's been awhile since I've done CSS. I see you are using `rem` units above and `px` units here. When is it appropriate to use which type of unit?

Reply:
I try to use rems when I can since rem is a relative unit measure so it's better accessibility wise, but I use px for the smaller adjustments (like the submit button had to align with the form text boxes)